scripts/apt-test.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').config()
+
+const { upload } = require('../src/api')
+
+const {
+  BUILD_PATH,
+  PLATFORM,
+  TOKEN,
+  URL,
+} = process.env
+
+if (!BUILD_PATH || !PLATFORM || !TOKEN || !URL)
```

Comment:
This should check for `!BUILD_PATH && !URL`
Technically only one of them is required

Updated to look like this and it works => 
`if ((!BUILD_PATH && !URL) || !PLATFORM || !TOKEN)`

Reply:
Ah, this was a last minute update that I messed up. My bad. Thanks, it's been updated.

Reply:
Also, yes, this is planned to be published to NPM. I just updated it to have a "repository" and a scoped "name" field, so it should be good to go once it's approved.